tokio-threadpool/src/lib.rs

```diff
@@ -1090,7 +1092,6 @@ impl Inner {
         trace!("worker_terminated; num_workers={}", prev - 1);
 
         if 1 == prev {
-            trace!("notifying shutdown task");
```

Comment:
Er, woops, did you want to keep this? I was just cleaning up a bunch of logs I had used to debug this, and thought this was one I had added.